integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -568,7 +568,7 @@ class CarbonFileMetastore extends CarbonMetaStore {
         val tableLocation = catalogTable.storage.locationUri match {
           case tableLoc@Some(uri) =>
             if (tableLoc.get.isInstanceOf[URI]) {
```

Comment:
still need to check isInstanceOf[URI] ? we are not using it now

Reply:
Done.